src/genai/schema/_api.py

```diff
@@ -100,8 +100,12 @@ class LengthPenalty(ApiBaseModel):
 
 
 class ModelFamily(ApiBaseModel):
+    description: Optional[str] = None
```

Comment:
This needs to be rebased after SystemPrompt service is merged.

Also, I generated the schema against dev, so you can see some changes like this one, I can put them into a separate PR if necessary :)

Also, I'd rewrite casettes just to be sure tests work with these changes, I'll add a separate commit with new casettes after review.